src/stirling/source_connectors/perf_profiler/BUILD.bazel

```diff
@@ -53,9 +53,7 @@ pl_cc_test(
         "//src/stirling/source_connectors/perf_profiler/testing/java:graal-vm-aot-fib",
     ],
     defines = ['JDK_IMAGE_NAMES=\\"%s\\"' % ",".join(jdk_image_names)],
-    shard_count = 3,
     tags = [
-        "cpu:16",
```

Comment:
Did you mean to change these? I thought we wanted to keep these in place.

Reply:
That crept in. Fixing it now. (thank you!)